test/Microsoft.Framework.Logging.Test/LoggerStructureFormatTest.cs

```diff
@@ -0,0 +1,61 @@
+﻿// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Linq;
+using Xunit;
+
+namespace Microsoft.Framework.Logging.Test
+{
+    public class LoggerStructureFormatTest
```

Comment:
Can you add tests for un-balanced cases? E.g. various bad strings such as `{foo` and `bar}`.